### PR TITLE
mentors can set tip of the round

### PR DESCRIPTION
**Pull request by GenericDM**
mentors can now set tip of the round

completely untested ain't that funny
#### Changelog
also it moves the set tip to mentor tab
:cl:  
rscadd: mentor can now set tip of the round
/:cl:

### CODE REVIEW

**Comment by maxion12345** [comment]
tip of the round might actually be useful

**Comment by boodaliboo** [comment]
i will mentorhelp everyround so they say my tip

**Comment by GenericDM** [comment]
just realized that i didnt even add this to mentor verbs... time to fix

**Comment by jcatjmeow** [comment]
Proc isn't documented

File: yogstation/code/modules/mentor/mentor_verbs.dm [hunk 1]
```diff
@@ -69,3 +70,29 @@ GLOBAL_PROTECT(mentor_verbs)
...
+/client/proc/show_tip()
+	set category = "Mentor"
+	set name = "Show Tip"
+	set desc = "Sends a tip (that you specify) to all players. After all \
+		you're the experienced player here."
+
+	if(!is_mentor())
+		return
+
+	var/input = input(usr, "Please specify your tip that you want to send to the players.", "Tip", "") as message|null
```
Comment (alexkar598) [inline yogstation/code/modules/mentor/mentor_verbs.dm:82]: Going to have to be sanitized

File: yogstation/code/modules/mentor/mentor_verbs.dm [hunk 2]
```diff
@@ -69,3 +70,29 @@ GLOBAL_PROTECT(mentor_verbs)
...
+		return
+
+	if(!SSticker)
+		return
+
+	SSticker.selected_tip = input
+
+	// If we've already tipped, then send it straight away.
+	if(SSticker.tipped)
+		SSticker.send_tip_of_the_round()
```
Comment (alexkar598) [inline yogstation/code/modules/mentor/mentor_verbs.dm:93]: Yeah no, no giving mentor an announce button

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by GenericDM** [comment]
i dont have the time to fix this for standards, maybe later

**Comment by Jarod1200** [comment]
f

**Pull request closed without merging**